examples/pom.xml

```diff
@@ -97,13 +97,10 @@
 
     <modules>
         <module>authentication</module>
-        <module>byteman</module>
```

Comment:
I can see that the jkube example is missing (could we add it in your PR?)

Reply:
Do you know by any chance if there is a simple way to skip the openshift-maven-plugin build/resource goals? they fails by default because I don't have OpenShift running at the moment

Reply:
-Djkube.skip should work.

Reply:
Thanks, that finally worked